src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -21,3 +21,7 @@ databaseChangeLog:
       file: db/changelog/changes/10-create-categories-table.yaml
   - include:
       file: db/changelog/changes/11-create-book_category-table.yaml
+  - include:
+      file: db/changelog/changes/12-create-shopping_carts-table.yaml
+  - include:
+      file: db/changelog/changes/13-create-cart_items-table.yaml
```

```suggestion
      file: db/changelog/changes/13-create-cart_items-table.yaml
      
```
forgot an empty line